neptune/alpha/experiment.py

```diff
@@ -133,6 +133,9 @@ def set_attribute(self, path: str, attribute: Attribute) -> Optional[Attribute]:
         with self._lock:
             return self._structure.set(parse_path(path), attribute)
 
+    def exists(self, path: str) -> bool:
+        return self.get_attribute(path) is not None
```

Comment:
Suggestion to define `__bool__` instead.

Reply:
__bool__ don't take arguments and is less intuitive for users.